employee-rostering/src/main/java/employeerostering/employeerostering/EmployeeRoster.java

```diff
@@ -86,23 +93,34 @@ public void setShiftAssignmentList(
     }
 
     public void setDayOffRequestList(
-            java.util.List<employeerostering.employeerostering.DayOffRequest> dayOffRequestList ) {
+            java.util.List<employeerostering.employeerostering.DayOffRequest> dayOffRequestList) {
         this.dayOffRequestList = dayOffRequestList;
     }
 
+    public HardSoftScore getScore() {
+        return score;
+    }
+
+    public void setScore(
+            org.optaplanner.core.api.score.buildin.hardsoft.HardSoftScore score) {
```

Comment:
Non-blocking comment (go ahead with merge):
A bit strange that CourseSchedule.java imports HardSoftScore and EmployeeRoster.java (this file) doesn't but uses full classnames instead.